indexer-app/src/cmr/indexer/data/concepts/collection.clj

```diff
@@ -173,6 +173,12 @@
        distinct
        (remove nil?)))
 
+(defn- cloud-hosted?
+  "Test if the collection meets the criteria for being cloud hosted"
+  [collection tags]
+  (or (not (empty? (:DirectDistributionInformation collection)))
+      (tag/has-cloud-s3-tag tags)))
```

Comment:
If this search param is meant to let us move away from using tags, why use them as part of the `cloud-hosted` predicate?

Reply:
because there will be a transition period. the people writing the tags need time to convince metadata providers to add the info to the URL.

Reply:
Gotcha - just wanted to confirm my suspicions 🙂

Reply:
I'm guessing we need a search functionality that returns all the cloud-holding collections but we have to gradually move towards tag usage to the DirectDistributionInformation URL..